packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -46,7 +59,7 @@ class Trade extends React.Component {
             this.setState({ digits: [] });
         }
         if (prevProps.symbol !== this.props.symbol) {
-            this.setState({ try_synthetic_indices: false });
+            this.setState({ try_synthetic_indices: false, try_open_markets: false });
```

Comment:
Setting the state onUnmount has no effect. We should avoid it.

Reply:
Could you elaborate? We're in `componentDidUpdate`

Reply:
Oh :joy: